lib/client.ts

```diff
@@ -1,7 +1,12 @@
 import { AtpAgent, AtpServiceClient, AtpSessionData } from '@atproto/api'
+import { AuthState } from './types'
+import { OzoneConfig, getConfig } from './client-config'
 
-interface ClientSession extends AtpSessionData {
+export interface ClientSession extends AtpSessionData {
   service: string
+  // @TODO consider backwards compat of local storage state
```

Comment:
I think this will cause an error on setup and the local state wont have a chance to fix itself, so I'm going to handle it in `_setup()` as you described 👍

Reply:
Actually I will add a check for it in `_loadSession()`, that will log the mods out but it's a bit cleaner.